client/src/components/ProductCard.js

```diff
@@ -59,31 +58,32 @@ export default function ProductCard(props) {
         })
         .then(response => {
             if (response.ok) {
-              return response;
+                return response;
             } else {
-              var error = new Error('Error ' + response.status + ': ' + response.statusText);
-              error.response = response;
-              throw error;
+                var error = new Error('Error ' + response.status + ': ' + response.statusText);
+                error.response = response;
+                throw error;
             }
             },
             error => {
-                    var errmess = new Error(error.message);
-                    throw errmess;
+                var errmess = new Error(error.message);
+                throw errmess;
         })
         .then(response => response.json())
-        .then(res=>{shListsContext.hideProduct(props.product);
+        .then(res=>{
+            shListsContext.hideProduct(props.product);
             shListsContext.handleProductDeletion(props.listId,props.product._id);})
-        .catch(error =>console.log('product-deleting failed', error.message));
+        .catch(error =>alert('product-deletion failed', error.message));
```

Comment:
Alert is fine for now. I think we should use a better error displaying method. There is a good library which I have used before - `connect flash` - https://www.npmjs.com/package/connect-flash

Reply:
I agree. I don't like `alert`.